src/com/company/NumbersOfPassengers.java

```diff
@@ -0,0 +1,7 @@
+package com.company;
+
+public class NumbersOfPassengers extends Exception{

```

Comment:
Please, add "Exception" postfix and remove the extra "Of". You can call your class like PassengersNumberException. Try to avoid the word "Of" in classes names.